diff --git a/node_modules/serverless-nextjs-plugin/lib/build.js b/node_modules/serverless-nextjs-plugin/lib/build.js
index 2498e9b..4e6a8d7 100644
--- a/node_modules/serverless-nextjs-plugin/lib/build.js
+++ b/node_modules/serverless-nextjs-plugin/lib/build.js
@@ -35,7 +35,7 @@ module.exports = async function() {
   servicePackage.include = servicePackage.include || [];
   servicePackage.include.push(
     path.posix.join(pluginBuildDir.posixBuildDir, "**"),
-    path.posix.join("node_modules/next-aws-lambda", "**")
+    path.posix.join("../node_modules/next-aws-lambda", "**")
   );
 
   const { nextConfiguration } = await parseNextConfiguration(nextConfigDir);